src/stylesheets/Book.css

```diff
@@ -0,0 +1,75 @@
+.Book {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  padding: 20px 30px;
+  border-radius: 5px;
+  margin: 10px 0 10px 0;
+}
+
+.book-actions span {
+  margin-right: 5%;
+  padding-right: 5%;
+  border-right: 1px solid lightgray;
+}
+
+.book-details {
+  flex: 40%;
+}
+
+.title,
+.category,
+.author {
+  margin: 0;
+}
+
+.category {
+  margin: 5px;
+  font-weight: bolder;
+}
+
+.page-details {
+  flex: 30%;
+  padding-left: 5%;
+}
+
+.page-details p {
+  margin: 0;
+}
+
+.progress-bar {
+  width: 4.25rem;
+  height: 4.25rem;
+  border: solid 5px #e8e8e8;
+  border-radius: 50%;
+  border-top-color: #4386bf;
+  border-right-color: #4386bf;
+}
+
+.progess-details {
+  flex: 15%;
+  display: flex;
```

Comment:
- [ ] [REQUIRED] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule.